### ISSUE TITLE
BrainTree can not build on Xcode8.1

### ISSUE THREAD

**Issue by robyzhou** [braintree/braintree_ios#277]
I have integrated BrainTree in my project through cocoapods, and I can build it on Xcode 8.0, but Xcode 8.1. See the snapshot bellow:

![a5aa9c4e-1200-473b-9c2f-43a8dc9de30b](https://cloud.githubusercontent.com/assets/6167223/19959454/f4519f72-a1e1-11e6-8aa7-e94fdab8efb5.png)

I think maybe the naming of bundle in BrainTree is not proper on Xcode 8.1, because you use '-' in the bundle file name. I have tried to degrade Xcode to version 8.0, and it works ok.

So, could you please try to fix this issue? Thanks a lot!

**Comment by skunkworks** [comment]
Hi @robyzhou, can you confirm which version of CocoaPods you're using (`pod --version`)? There is a good chance this is occurring because you're on a pre-major release version (<1.0).

**Comment by robyzhou** [comment]
@skunkworks Hi, I am using version 0.39.0. So you think it's a CocoaPods issue?

**Comment by demerino** [comment]
Your `Cocoapods` version could be the root problem. If you still can't figure out how to update to `Cocoapods 1.1.1` - you might want to look into Carthage. 

Based on our tests, `Braintree` works with Xcode8.X with current Cocoapod and Carthage integrations. In similar situations, I've noticed that sometimes you'll need to clear your `derived data` and blow away your `/Pods` folder followed by a `pod update`.

Let us know if you still need help!

**Comment by alphahelixius** [comment]
Hi, I encountered this issue with Xcode8.1 as well. Is it tested for Xcode8.1? It was working before the update.

**Comment by robyzhou** [comment]
Yes, I have tested it on Xcode 8.1. I am trying to upgrade my CocoaPods to the latest version as suggested by @skunkworks and @demerino

**Comment by demerino** [comment]
I am testing in Xcode8.1 (using Cocoapods 1.1.1). For reference, the Podfile looks like this for my sample ObjC project:

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'OBJCButtonStore' do
  pod 'Braintree'
end
```

**Comment by skunkworks** [comment]
I have confirmed that updating from CocoaPods 0.39.0 to 1.1.1 will eliminate this error on Xcode 8.1. Please let us know if running `gem install cocoapods` fixes this issue for you. Thanks!

**Comment by alphahelixius** [comment]
Verified that Cocoapods 1.1.1 works for Xcode 8.1 for me. Thanks.

**Comment by robyzhou** [comment]
Confirm again, upgrade CocoaPods to v1.1.1 will make Braintree works well on Xcode8.1. Thanks a lot @skunkworks @demerino

**Comment by vemahendran** [comment]
@skunkworks @demerino Our app can only run on CocoaPods 0.39.0. We have a hard dependency with the pod version. It would be really great if you guys fix this issue on 0.39.0 too?
